RELEASE_NOTES

```diff
@@ -6,9 +6,6 @@
 
  * Build Issues
    - CH4 will not build with Solaris compilers.
- * Stability with large buffers
-   - CH4 may suffer from stability issues when operating on large
-     communication buffers (>= 2GB)
 
  * Dynamic process support
    - Requires fi_tagged capability in OFI netmod
```

Comment:
Dynamic process does not work with ucx, right?

Reply:
Yes, I should add a note about that.